devel/capnproto/Portfile

```diff
@@ -4,9 +4,10 @@ PortSystem              1.0
 PortGroup               compiler_blacklist_versions 1.0
 
 name                    capnproto
-version                 0.6.1
-checksums               rmd160  0e8efcdb45e2d4da6d800d82bf7aa4e5225b3054 \
-                        sha256  8082040cd8c3b93c0e4fc72f2799990c72fdcf21c2b5ecdae6611482a14f1a04
+version                 0.7.0
```

Comment:
Have you tested building `textmate2` with this change? I'm guessing the `textmate2` is currently broken anyway, though, so it's probably a non-issue.

Reply:
I can patch textmate2 when it won't have other issues and capnp will be the only blocker (it may not be a blocker by then, say).